src/getCoinCombination.test.js

```diff
@@ -1,10 +1,68 @@
 'use strict';
 
-describe('getCoinCombination', () => {
-  const { getCoinCombination } = require('./getCoinCombination');
+const { getCoinCombination } = require('./getCoinCombination.js');
 
+describe(`test getCoinCombination function`, () => {
   it(`should be declared`, () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it(`'getCoinCombination' should return`
+  + `error message with Negative number of cents`, () => {
+    expect(() => getCoinCombination(-30)).toThrow();
+  });
+
+  it(`'getCoinCombination' should return`
+  + `error message if cents = 0`, () => {
+    expect(() => getCoinCombination(0)).toThrow();
```

Comment:
The test case for when `cents = 0` should not expect an error to be thrown. According to the task description, the function should handle non-negative integers, and zero is a valid input that should return an array of zeros.